Utils/DashXUtils.swift

```diff
@@ -10,19 +10,32 @@ import DashX
 
 class DashXUtils {
     // Client using shared/default instance
-    static let client2 = getClient2()
+    static let client1 = getClient()
     
-    // Client using explicit instance
-    static let client1 = DashXClient(withPublicKey: "R7De1FaPwj6JkG1aDICbO82V",
-                                    baseURI: "https://api.dashx-staging.com/graphql")
-    
-    static func getClient2() -> DashXClient {
-//        DashX.setPublicKey(to: "R7De1FaPwj6JkG1aDICbO82V")
-//        DashX.setBaseURI(to: "https://api.dashx-staging.com/graphql")
-        
+    static func getClient() -> DashXClient {
+        DashX.setPublicKey(to: "Obmgjd5Ail9L1UBzI7RpPz2Y")
+        DashX.setBaseURI(to: "https://api.dashx-staging.com/graphql")
+        DashX.setTargetEnvironment(to: "staging")
         return DashX
```

Comment:
Can we get rid of this DashXUtils class? It feels like this is not showing a good way to integrate with DashX.